cmd/snap/cmd_snap_op.go

```diff
@@ -970,8 +977,9 @@ type cmdRevert struct {
 	waitMixin
 
 	modeMixin
-	Revision   string `long:"revision"`
-	Positional struct {
+	Revision      string `long:"revision"`
+	IgnoreRunning bool   `long:"ignore-running"`
```

Comment:
I think this should start hidden.  The name is reasonable but we will also add a way for snap to have a hook that can block refreshes, that might need  yet another flag,  I'm a bit unsure ATM whether we want then two flags or a single "force" flags that covers more.

Reply:
I agree with the rationale, done.